examples/09.javascript/bytecode.inc

```diff
@@ -1 +1 @@
-{0x07,0x1c,0x07,0x00,0x16,0x01,0xd7,0x60,0x03,0x00,0x00,0x00,0x1c,0x00,0x1e,0x00,0x22,0x00,0x22,0x00,0x28,0x00,0x32,0x00,0xf4,0x00,0x02,0x01,0x01,0x00,0xd2,0x04,0xad,0x00,0x01,0x01,0xfd,0x00,0x01,0x00,0x49,0x00,0x65,0x00,0x59,0x00,0x3d,0x00,0x35,0x00,0x05,0x40,0x70,0x75,0x73,0x68,0x00,0x00,0x04,0x40,0x6c,0x6f,0x67,0x00,0x00,0x00,0x02,0x60,0x00,0x00,0x0e,0x40,0x48,0x65,0x6c,0x6c,0x6f,0x2c,0x20,0x57,0x6f,0x72,0x6c,0x64,0x21,0x00,0x07,0x40,0x61,0x72,0x72,0x61,0x79,0x5b,0x00,0x00,0x00,0x00,0x05,0x40,0x5d,0x20,0x3d,0x20,0x00,0x00,0x0d,0x50,0x04,0x31,0x30,0x30,0x88,0x1d,0x00,0x6b,0x12,0x6f,0x67,0x01,0x60,0x00,0x2b,0x50,0x06,0x01,0x01,0x7e,0x00,0xa1,0x06,0xa0,0x10,0x89,0x02,0x00,0xe0,0x70,0x02,0x11,0x60,0x89,0x01,0x00,0x08,0x6c,0x10,0x8c,0x01,0x00,0x67,0x89,0x01,0x00,0x12,0x12,0x12,0x6f,0x67,0x10,0x07,0x6c,0x10,0xa1,0x67,0x76,0xdd,0x00,0x00,0x00,0x48,0x50,0x0a,0x01,0x01,0x89,0x00,0x00,0x10,0x88,0x3d,0x00,0x6b,0x11,0x88,0x49,0x00,0x78,0x02,0xa0,0x67,0x89,0x03,0x00,0x01,0x78,0x01,0xa1,0x06,0xa0,0x06,0x10,0xa1,0x67,0x10,0x89,0x02,0x00,0xe0,0x70,0x02,0x01,0x60,0x89,0x00,0x00,0x10,0x88,0x3d,0x00,0x6b,0x11,0x88,0x59,0x00,0x14,0x88,0x65,0x00,0x17,0x17,0x6b,0x78,0x05,0xa0,0x67,0x10,0x07,0x6c,0x10,0xa1,0x67,0x76,0xd8,0x0c,0x00,0x03,0x00,0x17,0x00,0x7d,0x00,0x02,0x00,0x19,0x00,0x01,0x00,0x08,0xc0,0x05,0x00,0x05,0x00,0x35,0x00,0x6d,0x00,0x08,0xc0,0x05,0x00,0x05,0x00,0x3d,0x00,0x45,0x00}
+{0x08,0x1c,0x00,0x00,0x1a,0x01,0x7d,0xe8,0x03,0x00,0x00,0x00,0x1c,0x00,0x1e,0x00,0x22,0x00,0x22,0x00,0x30,0x00,0x3a,0x00,0xf6,0x00,0x06,0x01,0x01,0x00,0xd2,0x04,0xb1,0x00,0x05,0x01,0x01,0x01,0x01,0x00,0x01,0x00,0x01,0x00,0x01,0x00,0x01,0x00,0x51,0x00,0x6d,0x00,0x61,0x00,0x45,0x00,0x3d,0x00,0x05,0x40,0x70,0x75,0x73,0x68,0x00,0x00,0x04,0x40,0x6c,0x6f,0x67,0x00,0x00,0x00,0x02,0x60,0x00,0x00,0x0e,0x40,0x48,0x65,0x6c,0x6c,0x6f,0x2c,0x20,0x57,0x6f,0x72,0x6c,0x64,0x21,0x00,0x07,0x40,0x61,0x72,0x72,0x61,0x79,0x5b,0x00,0x00,0x00,0x00,0x05,0x40,0x5d,0x20,0x3d,0x20,0x00,0x00,0x04,0x50,0x31,0x30,0x30,0x88,0x1d,0x00,0x6b,0x12,0x6f,0x67,0x01,0x60,0x00,0x00,0x06,0x50,0x01,0x01,0x7e,0x00,0xa1,0x06,0xa0,0x10,0x89,0x02,0x00,0xe0,0x70,0x02,0x11,0x60,0x89,0x01,0x00,0x08,0x6c,0x10,0x8c,0x01,0x00,0x67,0x89,0x01,0x00,0x12,0x12,0x12,0x6f,0x67,0x10,0x07,0x6c,0x10,0xa1,0x67,0x76,0xdd,0x09,0x50,0x01,0x01,0x01,0x89,0x00,0x00,0x10,0x88,0x45,0x00,0x6b,0xa1,0x88,0x51,0x00,0x78,0x82,0x89,0x03,0x00,0x01,0x78,0x01,0xa1,0x06,0xa0,0x06,0x10,0xa1,0x67,0x10,0x89,0x02,0x00,0xe0,0x70,0x02,0x01,0x60,0x01,0x89,0x00,0x00,0x10,0x88,0x45,0x00,0x6b,0xa1,0x88,0x61,0x00,0x13,0x88,0x6d,0x00,0x16,0x16,0x6b,0x78,0x85,0x10,0x07,0x6c,0x10,0xa1,0x67,0x76,0xd9,0x00,0x0c,0x00,0x03,0x00,0x17,0x00,0x85,0x00,0x19,0x00,0x02,0x00,0x19,0x00,0x01,0x00,0x08,0xc0,0x05,0x00,0x05,0x00,0x3d,0x00,0x75,0x00,0x08,0xc0,0x05,0x00,0x05,0x00,0x45,0x00,0x4d,0x00}
```

Comment:
I'm just going to trust that that's correct, since I am not a cylon.

Reply:
> I am not a cylon

I will defer fixing that for a later PR.